turbinia/turbiniactl.py

```diff
@@ -388,6 +389,20 @@ def main():
       'Additionaly, you can use the -a or --all_fields flag to retrieve the '
       'full output containing finished and unassigned worker tasks.')
 
+  # Add GCS logs collector
+  parser_gcs_logs = subparsers.add_parser(
+      'DumpGCSLogs', help='Get Turbinia results from Google Cloud Storage.')
+  parser_gcs_logs.add_argument(
+      '-p', '--gcs_path',
+      help='GCS bucket to pull logs from. Must be in the following format gs://{BUCKET_NAME}/{OUTPUT_DIR}',
+      required=False)
+  parser_gcs_logs.add_argument(
+      '-o', '--output_dir', help='Directory path for output.', required=True)
+  parser_gcs_logs.add_argument(
```

Comment:
Rather than matching on arbitrary file names here and searching every file, maybe we can have flags for `--request` and possible `--task` and then look up the exact paths in datastore?